.github/pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+Implements [DHIS2-XXXX](https://dhis2.atlassian.net/browse/DHIS2-XXXX)
+
+---
+
+### Key features
+<!-- Remove if not applicable -->
+1. _feature_
+
+---
+
+### Description
+
+_text_
+
+---
+
+### BREAKING CHANGE
+<!-- Remove if not applicable -->
+_reason_
+
+---
+
+### Checklist
+
+-   [ ] Write Documentation
+    - _If not needed, explain why, otherwise remove this bullet point_
+-   [ ] Write Tests
+    - _If not needed, explain why, otherwise remove this bullet point_
+-   [ ] Actual task completed
+---
+
+### Known issues
+<!-- Remove if not applicable -->
+-   [ ] _issue_
```

Comment:
This is perhaps a bit neurotic of me, but I prefer not to use a checkbox here. I can try to rationalise it by saying: it's a known issue, so by definition it's not getting fixed in this PR, and therefore we should show a bullet point instead of a checkbox.

But really it is all about this (see "1 of 3 tasks")....
<img width="482" alt="Screenshot 2023-02-09 at 15 59 42" src="https://user-images.githubusercontent.com/353236/217848898-88a6fe0a-a246-45c2-a1ea-90756118fe1a.png">
Every PR with known issues as checkboxes would look incomplete in the GitHub UI 😱